modules/server_new/src/main/scala/observe/server/SeqTranslate.scala

```diff
@@ -777,39 +777,39 @@ object SeqTranslate {
 //    private val gnirsDisabled: GnirsController[F]           = new GnirsControllerDisabled[F]
 
     def tcsSouth(overrides: SystemOverrides): TcsSouthController[F] =
-      if (overrides.isTcsEnabled) systems.tcsSouth
+      if (overrides.isTcsEnabled.value) systems.tcsSouth
       else tcsSouthDisabled
 
     def tcsNorth(overrides: SystemOverrides): TcsNorthController[F] =
-      if (overrides.isTcsEnabled) systems.tcsNorth
+      if (overrides.isTcsEnabled.value) systems.tcsNorth
       else tcsNorthDisabled
 
     def gems(overrides: SystemOverrides): GemsController[F] =
-      if (overrides.isTcsEnabled) systems.gems
+      if (overrides.isTcsEnabled.value) systems.gems
       else gemsDisabled
 
     def altair(overrides: SystemOverrides): AltairController[F] =
-      if (overrides.isTcsEnabled) systems.altair
+      if (overrides.isTcsEnabled.value) systems.altair
```

Comment:
Ah, I see here that the boolean value is used. We could check for ` === Enabled` instead.

Anyway, I don't see clear advantages or disadvantages of either method.